wnprc_billing/resources/module.xml

```diff
@@ -17,18 +17,18 @@
             <defaultValue>Please set module property BillingAddress</defaultValue>
         </propertyDescriptor>
         <propertyDescriptor name="CreditToAccount">
-            <canSetPerContainer>false</canSetPerContainer>
+            <canSetPerContainer>true</canSetPerContainer>
             <description>This is the Credit account information that will appear in the invoice header.</description>
             <editPermissions>
-                <permission>ADMIN</permission>
+                <permission>UPDATE</permission>
             </editPermissions>
             <defaultValue>Please set module property CreditToAccount</defaultValue>
         </propertyDescriptor>
         <propertyDescriptor name="JETCSVSetting">
-            <canSetPerContainer>false</canSetPerContainer>
+            <canSetPerContainer>true</canSetPerContainer>
```

Comment:
just curious why this was turned on to set per container? is the setting different for different containers?

Reply:
Changing canSetPerContainer to true throws an AJAX error: "This property can not be set for this container.  It can only be set site-side, which means it must be set on the root container."

I brought this up to LabKey when I started this fix and I think their response was that you can only change site-wide module properties from "Admin > Folder Management > Module Properties".  To change a module property via AJAX response, it must be container-specific.

Also note, changing the permission from ADMIN to UPDATE is required to allow non-admin users to change the container-specific value via AJAX.